src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1986,7 +1986,7 @@ public function setIncrementing($value)
     /**
      * Get the value of the model's primary key.
      *
-     * @return mixed
+     * @return null|array-key
```

Comment:
generally, it's preferred for `null` to come last:



```suggestion
     * @return array-key|null
```